source/quick-start.txt

```diff
@@ -50,6 +50,13 @@ If you are using `Gradle <https://gradle.org/>`__, add the following to your
 
 .. include:: /includes/quick-start/gradle-versioned.rst
 
+.. note::
+   
+   We recommend that you use a package manager to install the
```

Comment:
Suggestion:
I think Maven and Gradle are more often described as "build tools" or "dependency managers" based on the specific part of the functionality that's being described.

For example:
- [Gradle Build Tool (newsletter page)](https://newsletter.gradle.org/)
- [Apache Maven 3.x description](https://maven.apache.org/ref/3.9.6/)
   > Maven is a project development management and comprehension tool. Based on the concept of a project object model: builds, dependency management, documentation creation, site publication, and distribution publication are all controlled from [the pom.xml declarative file](https://maven.apache.org/ref/3.9.6/maven-model/maven.html). Maven can be extended by [plugins](https://maven.apache.org/plugins/) to utilise a number of other development tools for reporting or the build process.